File: algorithms/1_easy/F/03731_find_missing_elements/sort.cpp
class Solution {
public:
    vector<int> findMissingElements(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        vector<int> res{};

        for (auto it{nums.cbegin() + 1}; it != nums.cend(); ++it) {
            for (int k{*(it - 1) + 1}; k < *it; ++k) {
                res.push_back(k);
            }
        }

        return res;
    }
};

File: algorithms/1_easy/F/03731_find_missing_elements/sort.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        nums.sort()
        res = []

        for i in range(1, len(nums)):
            for k in range(nums[i - 1] + 1, nums[i]):
                res.append(k)

        return res
